build/build.xml

```diff
@@ -277,6 +277,43 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
+		<exec dir="${dir}"
+			command='php -r "echo version_compare(&apos;${version}&apos;, &apos;3.2.0-b3&apos;, &apos;>=&apos;) ? &apos;true&apos; : &apos;false&apos;;"'
+			checkreturn="true"
+			outputProperty='viglink-available' />
+		<if>
+			<and>
+				<equals arg1="${add-viglink-ext}" arg2="1" trim="true" />
+				<or>
+					<equals arg1="${revision}" arg2="HEAD" trim="true" />
+					<equals arg1="${viglink-available}" arg2="1" trim="true" />
+				</or>
+			</and>
```

Comment:
Well, how else would you include it? We decide later if we update the files.. ;)